### ISSUE TITLE
Autosuggest missing `Aria-atomic="true"` on status container

### ISSUE THREAD

**Issue by jrbarnes9** [ONSdigital/design-system#2622]
Autosuggest missing `Aria-atomic="true"` on status container

Reported by Site Improve:
> "The page contains an error container made with the WAI-ARIA role="alert" or aria-live="assertive" attributes, but it does not contain a WAI-ARIA attribute that will make sure all assistive technologies can read the error message after more than one invalid submission.

> To make sure that all assistive technologies will read the error message after more than one invalid submission, you need to add the WAI-ARIA attribute aria-atomic="true" to the error container tag."

https://www.w3.org/TR/WCAG20-TECHS/ARIA19.html

_Originally posted by @jrbarnes9 in https://github.com/ONSdigital/design-system/discussions/894#discussioncomment-5055446_
